entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/model/EntitySite.java

```diff
@@ -167,6 +169,20 @@ public EntitySite(String title,
         getUserRoles(); // populate the user roles
     }
 
+    public EntitySite(Site site, boolean includeGroups, boolean justInfo) {
```

Comment:
I know, but it's just a constructor marker. I could just rename it to markerArgument or something. Thanks for having a look.